src/dom-to-image.js

```diff
@@ -137,8 +138,16 @@
             canvas.width = options.width || util.width(domNode);
             canvas.height = options.height || util.height(domNode);
 
+            var ctx = canvas.getContext('2d');
+
+            if (options.dpi != null) {
+              var scaleFactor = options.dpi / 96;
```

Comment:
Hi! Sorry for not responding that long.

What if we introduce, instead of `oprions.dpi`, `options.dpiScale` , which can be calculated by client code - just like line 141 `options.dpiScale = desiredDpi / currentDpi`.

That way we can avoid looking for the device dpi in lib or hardcoding it.